demos/smart_classroom_demo/main.cpp

```diff
@@ -95,32 +149,189 @@ class Visualizer {
                               bbox_color, cv::FILLED);
             }
             if (!label_to_draw.empty()) {
-                putText(frame_, label_to_draw, cv::Point(rect.x, rect.y), cv::FONT_HERSHEY_PLAIN, 1,
-                        text_color, 1, cv::LINE_AA);
+                cv::putText(frame_, label_to_draw, cv::Point(rect.x, rect.y), cv::FONT_HERSHEY_PLAIN, 1,
+                            text_color, 1, cv::LINE_AA);
             }
         }
     }
 
-    void DrawFPS(float fps) {
+    void DrawFPS(const float fps, const cv::Scalar& color) {
         if (enabled_ && !writer_.isOpened()) {
             cv::putText(frame_,
                         std::to_string(static_cast<int>(fps)) + " fps",
                         cv::Point(10, 50), cv::FONT_HERSHEY_SIMPLEX, 2,
-                        cv::Scalar(0, 0, 255), 2, cv::LINE_AA);
+                        color, 2, cv::LINE_AA);
+        }
+    }
+
+    void CreateTopWindow() {
+        if (enabled_ && num_top_persons_ > 0) {
```

Comment:
Please, negate this condition, and if the negated condition is true, make `return`

Reply:
Done